lib/components/user/notification-prefs-pane.js

```diff
@@ -123,7 +114,12 @@ const NotificationPrefsPane = ({
                 onChange={handleChange}
                 value={type}
               >
-                {text}
+                {type === 'email'
```

Comment:
Here especially I would move to template strings/add a note about this being cleaner under typescript